src/js/components/Layer/__tests__/Layer-test.js

```diff
@@ -526,4 +526,137 @@ describe('Layer', () => {
     );
     expectPortal('singleId-test').toMatchSnapshot();
   });
+
+  test('restore focus to trigger button after closing with Esc', (done) => {
+    jest.useFakeTimers();
+
+    const TriggerButtonTest = () => {
+      const [showLayer, setShowLayer] = React.useState(false);
+      return (
+        <div>
+          <button
+            type="button"
+            data-testid="trigger-button"
```

Comment:
nit pik can we consider using `getByRole` instead of `getByTestId` where possible.